docs/src/pages/component-demos/text-fields/TextFields.js

```diff
@@ -77,6 +77,22 @@ class TextFields extends Component {
           defaultValue="2017-05-24"
           className={classes.input}
         />
+        <TextField
+          id="helperText"
+          label="Helper text"
+          type="text"
+          defaultValue="Default Value"
+          className={classes.input}
+          helperText="Some important text"
+        />
+        <TextField
```

Comment:
Also, Firefox do not like that example:
![capture d ecran 2017-05-29 a 21 41 06](https://cloud.githubusercontent.com/assets/3165635/26560535/bd25db8e-44b7-11e7-9b36-0b1e3403e001.png)